evm/bridge-token-factory/contracts/Borsh.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+library Borsh {
```

Comment:
We must figure out how to share this `Brosh.sol` and `SelectivePausableUpgradable` with other contracts.
So we have two options here: put all the contracts under the same project but in different directories like it is done in `openzeppelin-contracts` repo; Or move the shared code to a separate project.

Reply:
Makes sense but we can think about it when we add another project